web/webkit/src/test/scala/net/liftweb/webapptest/snippet/DeferredSnippet.scala

```diff
@@ -40,10 +40,9 @@ class DeferredSnippet {
   }
 
    def stackWhack: NodeSeq = {
-     val inActor: Boolean = Thread.currentThread.getStackTrace.
-     find(_.getClassName.contains("net.liftweb.actor.")).isDefined
+     val inActor: Boolean = Thread.currentThread.getStackTrace.find(_.getClassName.contains("net.liftweb.actor.")).isDefined
```

Comment:
Whoops, shall we make this one last change? Other than that I'm ready to pull the trigger here I think.